### ISSUE TITLE
Improve light theme contrast ratio

### ISSUE THREAD

**Issue by jahirfiquitiva** [fabian-hiller/vscode-pace-theme#1]
Hey, I recently found this theme and I'm really enjoying it. Anyway, the light theme colors are a bit hard to read for me.

I checked the contrast ratio of them and they all fail in terms of accessibility.
I used [contraste](https://contrasteapp.com/) for checking that.

Would you mind updating the colors to a darker shade, or maybe creating a high contrast variant with colors easier to read?

Thanks in advance.

<img width="512" alt="Shot 2022-09-04 at 13 39 52@2x" src="https://user-images.githubusercontent.com/10360816/188328656-6f0cd8ef-a407-4b91-aacf-012062969c83.png">

**Comment by fabian-hiller** [comment]
Thank you very much for your feedback! I mainly use the dark theme, so I focused on that during development. I will take another look at the colors of the light theme. I will give you an update in the coming days.

**Comment by jahirfiquitiva** [comment]
@fabian-hiller awesome, thanks!

**Comment by fabian-hiller** [comment]
Hey @jahirfiquitiva, I added a Light+ theme with higher contrast. Once you update the theme to v0.2.0, it should show up in the list. Do you think the contrast is sufficient? Feel free to leave me feedback on it here!

**Comment by jahirfiquitiva** [comment]
@fabian-hiller thanks for the update. I checked and all colors contrast ratio still fail except for the red and purple shades.

Anyway, the texts are easier to read now, so it's definitely an improvement, thanks!

Btw, I found another issue with `unused` (?) variables or lines ... the text is even less visible and much harder to read 😕 it would also be great if you could take a look at it.

<img width="623" alt="Shot 2022-09-06 at 16 42 52@2x" src="https://user-images.githubusercontent.com/10360816/188746486-1ffcca4d-d0c6-457d-9dd1-f39414a21a6e.png">

**Comment by fabian-hiller** [comment]
Thanks again for the feedback! Would you recommend raising the contrast another notch? In this case, the colors will continue to lose some of their "color".

> Btw, I found another issue with unused (?) variables or lines ...

Thank you for pointing that out, of course it shouldn't be like that. I will have a look at it in the next few days.

**Comment by jahirfiquitiva** [comment]
@fabian-hiller yeah, I would recommend making the colors a bit darker. Another thing maybe you could try is not using white for the background for rather a light shade of gray like `#FAFAFA` or `#ECEFF1` 🤔

**Comment by fabian-hiller** [comment]
I made the colors a bit darker again and added a new theme called Light++. Feel free to give me feedback compared to the Light+ theme.

I also experimented with the background color. However, I didn't like the results, so I stuck with the white background. If you want to customize the theme to your needs, you can create a fork or just copy the code.

**Comment by jahirfiquitiva** [comment]
@fabian-hiller Thank you so much! it looks much better... do you think it would be possible to keep both Light+ and Light++? That would allow people to choose the one they like better or fits better to their visual conditions 🤔

**Comment by fabian-hiller** [comment]
Sorry for the late feedback. Until version 1.0, I'm open to experimentation. Can you think of a better name for the themes that draws attention to the difference in contrast?

**Comment by jahirfiquitiva** [comment]
@fabian-hiller i'm bad at naming things 😆 but maybe:
- Light+ -> High Contrast
- Light++ -> A11y Compliant
?

**Comment by fabian-hiller** [comment]
Let's keep for now the current names. Thanks again for your feedback! 🚀
